src/Stratis.FederatedPeg.Features.FederationGateway/FederationGatewaySettings.cs

```diff
@@ -45,7 +47,10 @@ public FederationGatewaySettings(NodeSettings nodeSettings)
             }
 
             this.SourceChainApiPort = configReader.GetOrDefault(SourceChainApiPortParam, 0);
-            this.FederationNodeIpEndPoints = configReader.GetOrDefault<string>(FederationIpsParam, null)?.Split(',').Select(a => a.ToIPEndPoint(nodeSettings.Network.DefaultPort));
+            this.FederationNodeIpEndPoints = configReader.GetOrDefault<string>(FederationIpsParam, null)?.Split(',')
+                .Select(a => a.ToIPEndPoint(nodeSettings.Network.DefaultPort));
+
+            this.MinimumDepositConfirmations = nodeSettings.Network.Consensus.MaxReorgLength + 1;

```

Comment:
Per my other comment, I think you meant to do a GetOrDefault here.

Reply:
yes initially I thought, but that we be super messy if some members are using one maturity and others another maturity !